agent/command/s3_copy.go

```diff
@@ -223,20 +232,6 @@ func (c *s3copy) copyWithRetry(ctx context.Context,
 			continue
 		}
 
-		if len(s3CopyFile.BuildVariants) > 0 && !utility.StringSliceContains(
-			s3CopyFile.BuildVariants, conf.BuildVariant.Name) {
-			continue
-		}
-
-		if ctx.Err() != nil {
-			return errors.New("s3copy operation received was canceled")
-		}
-
-		logger.Execution().Infof("Making API push copy call to "+
```

Comment:
I kind of think it still makes sense to log here since we've now officially "taken the lock" on the push log but whatever makes most sense for your debugging